core/services/bag_of_holding/main.py

```diff
@@ -55,17 +55,24 @@ def write_db(data: Dict[str, Any]) -> None:
 @app.post("/set/{path}")
 @version(1, 0)
 async def write_data(path: str, payload: dict[str, Any] = Body(...)) -> JSONResponse:
+    logger.debug(f'Write path: {path}, {json.dumps(payload)}')
     current_data = read_db()
-    dpath.new(current_data, path, payload, separator=".")
+    if path == 'root':
+        current_data = payload
+    else:
+        dpath.new(current_data, path, payload, separator=".")
     write_db(current_data)
```

Comment:
So.. Here we are replacing the entire db without checking if it's structure is correct. I can't evaluate the implications because I'm not aware of the rest of the code, I believe you judgement

Reply:
like.. a `write_data("/root")` call would erase the db

Reply:
I have created an endpoint to only do that, making it more explicit